client/components/DrawingGame.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import * as handTrack from 'handtrackjs';
+
+const modelParams = {
+  flipHorizontal: true,
+  // reduce input image size for gains in speed.
+  imageScaleFactor: 0.7,
+  // maximum number of boxes to detect
+  maxNumBoxes: 1,
+  iouThreshold: 0.5,
+  scoreThreshold: 0.75,
+};
+
+let model = null;
+const video = document.getElementById('myVideo');
+const canvas = document.getElementById('canvas');
+const context = canvas.getContext('2d');
+
+export default class DrawingGame extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isVideo: false,
+      message: '',
+    };
+    this.startVideo = this.startVideo.bind(this);
+    // this.toggleVideo = this.toggleVideo.bind(this);
+    this.runDetection = this.runDetection.bind(this);
+    this.startGame = this.startGame.bind(this);
+  }
+
+  startVideo() {
+    handTrack.startVideo(video)
+      .then((status) => {
+        console.log('video started', status);
```

Comment:
I'm sure you're already aware there are a few console.logs throughout this pull request. I won't note each one, and it's okay to keep them for testing, just wanted to note that we'll have to go back and clean them up

Reply:
I commented out the console.logs. I don't want to remove them all the way yet since I'm still working with Handtrack.js.